include/solvers/navier_stokes_scratch_data.h

```diff
@@ -724,11 +724,7 @@ class NavierStokesScratchData
       number_of_particles = i_particle;
     }
 
-    // If there were particles, calculate the average particle velocity within
-    // the cell
-    if (number_of_particles > 0)
-      average_particle_velocity =
-        average_particle_velocity / number_of_particles;
+
```

Comment:
```suggestion
```

Reply:
Well it's not the particle_id right, particles they have an id, they store it as a property, so this would be even more confusing because this particle_i or i_particle is a loop index.

As for removing the feature, this should be done in a seperate PR standing on its own with a CHANGELOG entry